src/utils/upload/processMultipartUpload.js

```diff
@@ -23,7 +26,9 @@ const putPartInS3 = async (blob, signedUrl, onUploadProgress, currentRetry = 0)
   }
 };
 
-const processMultipartUpload = async (file, signedUrls, createOnUploadProgressForPart) => {
+const processMultipartUpload = async (
+  file, signedUrls, createOnUploadProgressForPart, abortController = null,
```

Comment:
is there actually any place where this is called without the `abortController` ?